service/status-report/main.go

```diff
@@ -141,6 +151,16 @@ func main() {
 	if err != nil {
 		klog.Exitf("Failed to update mon configmap for storageClient %v: %v", storageClient.Status.ConsumerID, err)
 	}
+	joinSecret := &corev1.Secret{}
+	joinSecret.Name = "noobaa-remote-join-secret"
+	joinSecret.Namespace = operatorNamespace
+	joinSecret.Data = map[string][]byte{
+		"auth_token": []byte(noobaaAuthToken),
+		"mgmt_addr":  []byte(noobaaMgmtAddress),
+	}
+	if err = cl.Create(ctx, joinSecret); err != nil {
```

Comment:
2 things:
1. I believe we need to register the scheme here as it is part of the status reporter's job. We might also need back for it.
2. I am not sure this is the correct place to create the secret Because status reporters should report only reporting should not create things if this is decided then I am ok with. 
3. Would you like to try creating a secret every minute?

Reply:
Added to scheme and added rbac
Can you suggest a better way or place I can add for the secret creation?

Reply:
@rchikatw 
removed secret creation from status-reporter and added it to storageclient_reconciler